BERGSTATION/GenerateSounds_neu.pl

```diff
@@ -285,10 +285,10 @@
     }
     AddToSoundfile($soundfile_kmh, "both", 3);                  # km/h.
 
-    AddToSoundfile($soundfile_timePrefix, "both", 0);           # Es ist
-    AddToSoundfile($ws[$hour], "both", 0);                      # (Stunde)
-    AddToSoundfile($soundfile_timeInfix, "both", 0);            # Uhr
-    AddToSoundfile($ws[$min], "both", 3);                       # (Minute)
+    AddToSoundfile($soundfile_timePrefix, "phone", 0);           # Es ist
```

Comment:
I think the idea of telling the time was to make sure that you don't get stuck with the same audio file for a while and no one takes notice.

As long as we keep the current structure (decoupled file generation and playback) this can be a little dangerous. I will add an issue to fix this. Comments in the issue...